tex/chapters/04-Implementation.tex

```diff
@@ -272,7 +290,21 @@ \subsection{Testing}
 
 Axelrod-Python uses several external programmes to help with testing.
 Travis \cite{Travis} provides continuous integration testing, so all tests are run automatically whenever any changes to code are made.
+AppVeyor \cite{AppVeyor} offers a similar service, but specialises in testing on Windows operating systems.
 Coveralls \cite{Coveralls} calculates what proportion of the code is tested, and can highlight areas that are either untested or need improvement.
+
+\begin{figure}[htbp!]
+\includegraphics[width = \textwidth]{../img/screenshots/ScreenShot5.png}
+\caption{A status report for Travis, AppVeyor and Coveralls}
+\label{fig:screenshot5}
+\end{figure}
+
+Figure \ref{fig:screenshot5} shows that at the point the submission was included into the main code base, all tests where passing on Travis and AppVeyor.
+Also, Coveralls showed that the overall test coverage had increased slightly.
+
 Finally, Hypothesis \cite{Hypothesis3.6.1} is a library for property testing.
-It works by generating random data matching a specification and checking that a specified guarantee still holds in that case.
-If it finds an example where it doesn’t, it takes that example and cuts it down to size, simplifying it until it finds a much smaller example that still causes the problem \cite{HypothesisDocs}.
+It works by generating random data matching a specification, and checking that a specified guarantee still holds in that case.
+If it finds an example where the guarantee does not hold, it takes that example and cuts it down to size, simplifying it until it finds a much smaller example that still causes the problem \cite{HypothesisDocs}.
+% TODO Do you want to add something about documentation and doctesting too?
```

Comment:
That's equivalent to describing your code contribution as "we have it and it's awesome".

I suggest showing snippets of http://axelrod.readthedocs.io/en/latest/tutorials/further_topics/fingerprinting.html (both the rendered html and the rst, saying what rst is etc...).

Discuss the doctests (in the same way that you've discussed the unit tests).

You could also download a pdf version of the docs: http://readthedocs.org/projects/axelrod/downloads/pdf/latest and grab the one page corresponding to fingerprinting and include that as an appendix.

You could possibly also find a comment/requests on the PR (if one exists) about the documentation perhaps?